src/main/webapp/WEB-INF/views/PokedexManagement.jsp

```diff
@@ -0,0 +1,152 @@
+                                    </div>
+                                </div>
+                            </div>
+
+                            <div class="row">
+                                <div class="form-group col-md-12">
+                                    <div class="col-md-7">
+                                        <input type="submit"
+                                            value="{{!pokemonController.pokemon.id ? 'Add' : 'Update'}}"
+                                            class="btn btn-primary btn-sm" ng-disabled="myForm.$invalid">
+                                        <button type="button" ng-click="pokemonController.reset()"
+                                            class="btn btn-warning btn-sm" ng-disabled="myForm.$pristine">Reset
+                                            Form</button>
+                                    </div>
+                                </div>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+                <div class="panel panel-default">
+                    <!-- Default panel contents -->
+                    <div class="panel-heading"><span class="lead">Pokedex of Pokemon </span></div>
+                    <div class="tablecontainer">
+                        <table class="table table-hover">
+                            <thead>
+                                <tr>
+                                    <th>ID</th>
+                                    <th>Name</th>
+                                    <th>Attack</th>
+                                    <th>Defense</th>
+                                    <th width="20%"></th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <tr ng-repeat="u in pokemonController.pokemons">
+                                    <td><span ng-bind="u.id"></span></td>
+                                    <td><span ng-bind="u.name"></span></td>
+                                    <td><span ng-bind="u.attack"></span></td>
+                                    <td><span ng-bind="u.defense"></span></td>
+                                    <td>
+                                        <button type="button" ng-click="pokemonController.edit(u.id)"
+                                            class="btn btn-success custom-width">Edit</button> <button type="button"
+                                            ng-click="pokemonController.remove(u.id)"
+                                            class="btn btn-danger custom-width">Remove</button>
+                                    </td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            </div>
+
+            <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.4/angular.js"></script>
```

Comment:
move the angularjs library script tag to the <head>, also, bump version to at least 1.6.x